website/www/site/content/en/blog/beam-2.41.0.md

```diff
@@ -0,0 +1,127 @@
+---
+title:  "Apache Beam 2.41.0"
+date:   2022-08-TODO 9:00:00 -0700
+categories:
+- blog
```

Comment:
Hi, could be helpful to add Release category:
open pr for exisiting release docs: #22785

Reply:
Done, thanks!